packages/api/lambdas/publish-reports.js

```diff
@@ -76,10 +76,13 @@ async function publishPdrSnsMessage(
 async function handleExecutionMessage(eventMessage) {
   try {
     const executionRecord = await Execution.generateRecord(eventMessage);
-    return publishExecutionSnsMessage(executionRecord);
+    return await publishExecutionSnsMessage(executionRecord);
```

Comment:
😞 Yeah.... I should have caught that one

Reply:
It didn't used to matter because `publishSnsMessage` down the line already caught and logged any errors that may have been thrown posting to SNS.